.github/workflows/docker.yml

```diff
@@ -182,7 +151,7 @@ jobs:
             org.opencontainers.image.documentation=https://www.openproject.org/docs/installation-and-operations/installation/
             org.opencontainers.image.vendor=OpenProject GmbH
           tags: |
-            type=semver,pattern={{version}},value=${{ needs.setup.outputs.version }}
+            ${{ needs.setup.outputs.docker_tags }}
```

Comment:
@oliverguenther do you think it is an issue if we pass the same tags here as we do in the merge step later on? I'm not sure I fully understand the requirements here.